tgui/packages/tgui/interfaces/CassetteManager.jsx

```diff
@@ -0,0 +1,66 @@
+import { useBackend } from '../backend';
```

Comment:
not required but it would be nice if you used typescript instead

Reply:
does 735faed address what you mean?